CHANGELOG.md

```diff
@@ -2,9 +2,9 @@
 
 ## Latest updates
 
-You can check all of our changes from [Release Page](https://github.com/Ikuyadeu/vscode-R/releases)
+You can check all of our changes from [Release Page](https://github.com/REditorSupport/vscode-R/releases)
 
-## [2.0.0](https://github.com/Ikuyadeu/vscode-R/releases/tag/v1.8.0)
+## [2.0.0](https://github.com/REditorSupport/vscode-R/releases/tag/v1.8.0)
```

Comment:
version number typo?

Reply:
Is was previously auto generated url. Fixed.